source/fundamentals/logging.txt

```diff
@@ -245,8 +245,8 @@ project.
       ``logback.xml`` file does not have to be in a specific location, but it must
       be accessible from your classpath. 
 
-      The Logback framework defines the following log levels. You can
-      see the log levels listed below in order from most urgent to least
+      The Logback framework defines the following log levels. The
+      following lists the log levels in order from most urgent to least
```

Comment:
```suggestion
      The Logback framework defines the following log levels,
      ordered from most urgent to least
```

Nit: I don't think "the following lists..." is a clear subject. What/who is "the following"? Applies to both logging edits.